src/feature.rs

```diff
@@ -126,7 +126,7 @@ fn expand_scenario(
     /// [`Examples`]: gherkin::Examples
     #[allow(clippy::expect_used)]
     static TEMPLATE_REGEX: Lazy<Regex> =
-        Lazy::new(|| Regex::new(r"<(\S+)>").expect("incorrect Regex"));
+        Lazy::new(|| Regex::new(r"<([^>]+)>").expect("incorrect Regex"));
```

Comment:
But we allow `[ \t\r\n\f]` now, which doesn't seem to be good. I think `[^>\s]` is better here.